File: wx/restart.py
import os, sys

print("killing makeServer.py")
os.system("ps -ef | grep makeServer.py | grep -v grep | cut -c 9-15 | xargs kill -s 9")

if len(sys.argv) == 2 and sys.argv[1] == "shut":
    sys.exit(0)

print("restarting makeServer.py")
os.system("nohup python makeServer.py&")

File: ruby/restart.rb
#!/env/bin/ruby
# encoding: utf-8

app_name = "bingAPI.rb"

puts "killing #{app_name}"
system "ps -ef | grep 5969 | grep -v grep | cut -c 9-15 | xargs kill -s 9"

puts "restarting #{app_name}"
system "nohup ruby #{app_name}&"
